src/components/StatusButton.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  GestureResponderEvent,
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import Svg, { Circle, Path } from 'react-native-svg';
+
+type Param = {
+  title: string;
+  status: number;
```

Comment:
nie podoba mi się to, że status jest typu `number`. Numer 0 nic tak naprawdę nie mówi o statusie. Docelowo to powinno być mapowane na jakiś `union type`